### PR TITLE
Illegal Garbage

### PR DESCRIPTION
**Pull request by Shosoul**

### CODE REVIEW

**Comment by Shosoul** [comment]
Soon, I'll have to make the different shapes possible. But otherwise, here's something I can do for now.

**Comment by Shosoul** [comment]
Renamed to "Illegal Garbage", this PR now allows all types of illegal garbage to be used.

File: engine.lua [hunk 1]
```diff
@@ -2350,7 +2350,16 @@ function Stack.check_matches(self)
 
       self:enqueue_card(false, first_panel_col, first_panel_row, combo_size)
       if self.garbage_target and self.garbage_target.telegraph then
-        self.garbage_target.telegraph:push("combo", combo_size, metal_count, first_panel_col, first_panel_row, self.CLOCK)
+        if metal_count > 3 then
```
Comment (JamesVanBoxtel) [inline engine.lua:2353]: I noticed when I made a 6 combo with 3 shock blocks and 3 normal blocks I didn't get a metal garbage, just a 5 wide block. Pretty sure thats a break.

**Review by JamesVanBoxtel — CHANGES_REQUESTED** [review]
Haven't got to fully review this yet but started testing and looking through it. Found what I think is a break but overall its looking good

File: engine.lua (hunk 1 above)
Comment (Shosoul) [reply engine.lua:2353]: What would normally result from this?

**Comment by Shosoul** [comment]
Should be good now.

**Review by JamesVanBoxtel — APPROVED** [review]

**Pull request merged**